### DISCUSSION TITLE
Connecting to ASMX Soap service

### DISCUSSION THREAD

**Discussion by cbealperto08** [dotnet/maui#10052] [Q&A]
Hi,
Is the asmx service is not supported by .Net MAUI? Adding connected service shows this error.
![error](https://user-images.githubusercontent.com/70960549/189569159-3f0ab413-fadb-474a-bdd7-ee5730d0910b.JPG)

Microsoft Visual Studio Professional 2022 (64-bit) - Current
Version 17.3.3

**Comment by superdri** [comment]
Hi, I have some issue in my visual studio, can you help me? thanks
